csrankings-x.csv

```diff
@@ -409,3 +409,4 @@ Xusheng Xiao,Case Western Reserve University,http://engineering.case.edu/groups/
 Xutao Li,Harbin Institute of Technology,http://faculty.hitsz.edu.cn/lixutao,hPm6fo8AAAAJ
 Xuyun Nie,UESTC,https://www.is.uestc.edu.cn/teachers.do?id=1087,NOSCHOLARPAGE
 Xuyun Zhang,Macquarie University,https://researchers.mq.edu.au/en/persons/xuyun-zhang,wbF6HL8AAAAJ
+Xingyu Zhou 0001,Wayne State University,https://xingyuzhou.org/,AsTyRmwAAAAJ

```

Comment:
ECE, not CS. Please re-open with evidence that they meet the inclusion criteria.